.github/workflows/ci.yml

```diff
@@ -71,9 +71,7 @@ jobs:
       # In order to be able to build with flow-go and the relic tag, we need to
       # run its go generate target.
       - name: Make Flow Go's crypto
-        run: |
-          cd ./flow-go
-          make crypto_setup_gopath
+        run: make crypto_setup
```

Comment:
isn't this what `make crypto_setup_gopath` is already doing? Maybe @tarakby has some ideas for how to make this more reliable

Reply:
What I see now is the right way to import flow-go/crypto in a project, as described in the package Readme :
 - we can delete the commented line https://github.com/onflow/flow-archive/blob/master/go.mod#L278
 - what is the reason of deleting the line https://github.com/onflow/flow-archive/blob/master/crypto_build.sh#L18. It actually allows the crypto version to be synched with what's imported `go.mod` without entering it [manually](https://github.com/onflow/flow-archive/blob/master/crypto_build.sh#L6).
 
 
`crypto_setup_gopath` is a make target that is defined by the `flow-go` repo, but this is a different repo, so it doesn't have to use the same target (flow-go/crypto is an independent package, it doesn't require flow-go to work). 
 This said, what is the purpose of checking out `onflow/flow-go` in the CI job above?
 If it was just to run `crypto_setup_gopath`, we should be able to delete it.

Reply:
Done!